test/extensions/filters/http/gcp_authn/gcp_authn_filter_test.cc

```diff
@@ -38,6 +38,7 @@ constexpr char DefaultConfig[] = R"EOF(
         base_interval: 1s
         max_interval: 10s
       num_retries: 5
+    header_name: customheader
```

Comment:
What is this `header_name` pointing to in the config? You are adding the `TokenHeader token_header = 4;`, right?

Reply:
this was some leftover from a previous iteration I think. Removed now.